src/microbe_stage/ProcessSystem.cs

```diff
@@ -64,35 +64,34 @@ public ProcessSystem(Node worldRoot)
 
         foreach (var organelle in enumerated)
         {
-            foreach (var efficiencyInfo in efficiencies)
+            foreach (var process in organelle.RunnableProcesses)
             {
-                foreach (var processData in efficiencyInfo.Value.Processes)
+                var processData = CalculateProcessMaximumSpeed(process, biome);
+
+                // Find process inputs and outputs that use/produce ATP
+                // and that are performed by this organelle
+                // and add to totals
+                if (!organelle.Processes.ContainsKey(processData.Process.InternalName))
```

Comment:
I think that this check can be entirely removed, as it was added originally to guard against counting processes in organelles that don't do them, but now as we calculate the per process data in this loop, this should not be necessary.